node/src/types/block.rs

```diff
@@ -1834,6 +1834,73 @@ impl Item for BlockWithMetadata {
     }
 }
 
+#[derive(DataSize, Clone, Debug, PartialOrd, Ord, PartialEq, Eq, Hash, Serialize, Deserialize)]
+/// Wrapper around block and its deploys.
+pub struct BlockAndDeploys {
+    /// Block part.
+    pub block: Block,
+    /// Deploys.
+    pub deploys: Vec<Deploy>,
+}
+
+impl Display for BlockAndDeploys {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
```

Comment:
```suggestion
    fn fmt(&self, f: &mut Formatter<'_>) -> fmt::Result {
```

Reply:
Fixed in [ecf4e89](https://github.com/casper-network/casper-node/pull/3010/commits/ecf4e895ee12a1b1c67900fe7d519bc4ed9dc658).